tdrs-backend/tdpservice/users/migrations/0033_user_nextgen_xid.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.11 on 2022-03-10 21:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
```

Comment:
The sub we get is just a plain ol string. if there is a way to get a guid, its through a mechanisim we don't know about at present